### PR TITLE
Linting some files for golangci-lint bump 🦁

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Small linting changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: pkg/reconciler/pipelinerun/pipelinerun_test.go [hunk 1]
```diff
@@ -943,7 +943,7 @@ func TestReconcileWithTimeout(t *testing.T) {
 	// Check that the expected TaskRun was created
 	actual := clients.Pipeline.Actions()[0].(ktesting.CreateAction).GetObject().(*v1alpha1.TaskRun)
 	if actual == nil {
-		t.Errorf("Expected a TaskRun to be created, but it wasn't.")
+		t.Fatalf("Expected a TaskRun to be created, but it wasn't.")
```
Comment (vdemeester) [inline pkg/reconciler/pipelinerun/pipelinerun_test.go]: `Fatal` returns right away, otherwise, if `actual == nil`, the code will panic, on link 950.

**Comment by bobcatfish** [comment]
@vdemeester what does "golangci-lint bump" mean?

**Comment by bobcatfish** [comment]
ah is it for https://github.com/tektoncd/plumbing/pull/279 ?

**Comment by vdemeester** [comment]
@bobcatfish yes :)

**Pull request merged**